Sources/Turf/GeoJSON.swift

```diff
@@ -0,0 +1,411 @@
+import Foundation
+#if !os(Linux)
+import CoreLocation
+#endif
+
+public typealias FeatureIdentifierNumber = Numeric & Codable
+public typealias FeatureIdentifierString = StringProtocol & Codable
+
+public struct FeatureIdentifier<T: FeatureIdentifierNumber, U: FeatureIdentifierString>: Codable {
```

Comment:
Would it be possible to represent the identifier as an enumeration with associated values (with cases `number(Numeric)` and `string(StringProtocol)`)? I think that would be easier for a developer to understand than a struct with template parameters. Or would that be incompatible with Codable?

Reply:
Protocols with associated values are incompatible with Codable but enumerations with associated values should work. However, we'd lose some type safety due to the lack of support for raw values but it would still be an improvement.